tools/eventkit-generator/main.go

```diff
@@ -0,0 +1,54 @@
+package main
```

Comment:
Also update the linters / tests to include tools folder. You need to call `go test`, `golangci-lint` and `staticcheck` per `go.mod`.